source/reference/minio-mc/mc-ilm-tier.rst

```diff
@@ -93,14 +93,22 @@ Transition Permissions
 Object transition lifecycle management rules require additional permissions on the remote storage tier. 
 Specifically, MinIO requires the remote tier credentials provide read, write, list, and delete permissions.
 
-For example, if the remote storage tier implements AWS IAM policy-based access control, the following policy provides the necessary permission for transitioning objects into and out of the remote tier:
+For example, if the remote storage tier implements AWS IAM policy-based access control, the following policy provides the necessary permissions for transitioning objects into and out of the remote tier:
 
 .. literalinclude:: /extra/examples/LifecycleManagementUser.json
    :language: json
    :class: copyable
 
 Modify the ``Resource`` for the bucket into which MinIO tiers objects.
 
+.. admonition:: Avoid remote tier object versioning
```

Comment:
this is probably not very clear ? it is better to clearly state that we are talking about versioning in the tiering backend. Something like this maybe? `Avoid enabling versioning in the remote tier`

Reply:
I like yours better! Also did some minor edits in the paragraph below.